README.md

```diff
@@ -21,11 +21,11 @@ This client can be used in any PHP project and with any framework.
 
 Install the client with Composer:
 
-`composer require cyberfusion/cluster-api-client`
```

Comment:
This should be reverted as the package has still the same name, see composer.json:2.